apps/ui/components/application/ReservationUnitList.tsx

```diff
@@ -53,7 +56,8 @@ const ReservationUnitList = ({
   const [showModal, setShowModal] = useState(false);
 
   const form = useFormContext<ApplicationFormValues>();
-  const { clearErrors, setError, watch, setValue } = form;
+  const { clearErrors, setError, watch, setValue, formState } = form;
+  const { errors } = formState;
 
```

Comment:
This could be just `const { clearErrors, setError, watch, setValue, formState: { errors } } = form;`, right? :)

Reply:
yeah was trying to avoid using 7 lines for it because of prettier

Reply:
Gotcha. 👍